src/main/java/appeng/tile/crafting/TileMolecularAssembler.java

```diff
@@ -654,12 +631,29 @@ public boolean isActive()
 	{
 		return this.isPowered;
 	}
+	
+	
+	private class CraftingGridFilter implements IAEItemFilter
+	{				
+		@Override
+		public boolean allowExtract(IItemHandlerModifiable inv, int slot, int amount) 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '(' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)